docs/Integrating Your HMI/Vehicle Data/index.md

```diff
@@ -0,0 +1,169 @@
+# Vehicle Data
+
+Vehicle data can be exposed to app developers by creating a `VehicleInfo` component within your HMI. The primary RPCs used by this component are:
+
+* VehicleInfo.GetVehicleData - A request from Core to retrieve specific vehicle data items from the system.
+* VehicleInfo.SubscribeVehicleData - A request from Core to receive periodic updates for specific vehicle data items from the system.
+* VehicleInfo.UnsubscribeVehicleData - A request from Core to stop receiving periodic updates for specific vehicle data items from the system.
+* VehicleInfo.OnVehicleData - A notification from the HMI indicating that one or more of the subscribed vehicle data items were updated.
+
+More information regarding this component is available in the VehicleInfo section of the [HMI Documentation](https://smartdevicelink.com/en/guides/hmi/overview/)
```

Comment:
Even though you link to the hmi documentation, can you add some example requests/ responses? Does not have to be for all of the rpcs, but maybe just a getVehicleData request & response would be useful.